sdk/schemaregistry/azure-schemaregistry/tests/test_schema_registry.py

```diff
@@ -141,7 +142,7 @@ def test_schema_update(self, schemaregistry_endpoint, schemaregistry_group, **kw
     def test_schema_same_twice(self, schemaregistry_endpoint, schemaregistry_group, **kwargs):
         client = self.create_client(schemaregistry_endpoint)
         schema_name = self.get_resource_name('test-schema-twice')
-        schema_str = """{"namespace":"example.avro","type":"record","name":"User","fields":[{"name":"name","type":"string"},{"name":"age","type":["int","null"]},{"name":"city","type":["string","null"]}]}"""
+        schema_str = """{"namespace":"example.avro","type":"record","name":"User","fields":[{"name":"name","type":"string"},{"name":"favorite_number","type":["int","null"]},{"name":"favorite_color","type":["string","null"]}]}"""
```

Comment:
out of curiosity, was there a reason for this change?

Reply:
nice catch, I mis-updated the test, no need for change.
reverted